{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SuccessSuccessSuccessSuccess\n",
      "\n",
      "\n",
      "\n",
      "SuccessSuccess\n",
      "\n",
      "Success\n",
      "Success\n",
      "SuccessSuccess\n",
      "\n",
      "Success\n",
      "SuccessSuccess\n",
      "Success\n",
      "\n",
      "SuccessSuccess\n",
      "\n",
      "errorSuccessSuccess\n",
      "\n",
      "\n",
      "Success\n",
      "SuccessSuccessSuccess\n",
      "Success\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-311:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-361:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-312:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-316:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-307:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-309:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-315:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-320:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-387:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-345:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-339:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-313:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-314:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-328:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-386:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-336:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-318:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-322:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-329:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-321:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-335:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-319:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-327:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-326:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-349:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-367:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-371:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-352:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-338:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-330:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-337:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-334:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-348:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-317:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-346:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-331:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-323:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-358:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-310:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-347:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-388:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-365:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-333:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-308:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-360:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-332:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-366:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-384:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-375:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-377:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-364:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-369:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-383:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-376:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-325:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-344:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-341:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-368:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-353:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-354:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-396:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-363:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-374:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-359:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-356:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-324:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Success\n",
      "Success\n",
      "SuccessSuccess\n",
      "SuccessSuccess\n",
      "Success\n",
      "error\n",
      "\n",
      "\n",
      "Success\n",
      "SuccessSuccessSuccessSuccess\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "SuccessSuccessSuccess\n",
      "Success\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "SuccessSuccess\n",
      "SuccessSuccess\n",
      "SuccessSuccessSuccessSuccess\n",
      "\n",
      "SuccessSuccess\n",
      "Success\n",
      "\n",
      "\n",
      "\n",
      "Success\n",
      "Success\n",
      "SuccessSuccessSuccess\n",
      "Success\n",
      "SuccessSuccessSuccess\n",
      "\n",
      "\n",
      "\n",
      "SuccessSuccess\n",
      "\n",
      "Success\n",
      "\n",
      "SuccessSuccess\n",
      "Success\n",
      "Success\n",
      "SuccessSuccess\n",
      "Success\n",
      "\n",
      "\n",
      "SuccessSuccess\n",
      "Success\n",
      "\n",
      "\n",
      "\n",
      "SuccessSuccessSuccess\n",
      "\n",
      "SuccessSuccessSuccess\n",
      "\n",
      "\n",
      "\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "\n",
      "Success\n",
      "Success\n",
      "SuccessSuccess\n",
      "\n",
      "SuccessSuccess\n",
      "errorerror\n",
      "\n",
      "\n",
      "errorSuccess\n",
      "Success\n",
      "Successerror\n",
      "\n",
      "\n",
      "SuccessSuccess\n",
      "Success\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-362:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-378:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-343:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-351:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-306:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-380:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-357:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-381:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-350:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-370:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-373:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-391:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-382:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-393:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-390:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-340:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-355:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-379:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-342:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-394:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-372:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-392:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-389:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-385:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-395:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-400:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-409:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-403:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-404:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-405:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-411:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-398:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-401:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-412:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-417:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-397:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "error\n",
      "\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-399:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-410:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-416:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-408:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-407:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-402:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-438:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-414:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "SuccessSuccess\n",
      "Success\n",
      "error\n",
      "\n",
      "SuccessSuccess\n",
      "\n",
      "Success\n",
      "Success\n",
      "Success"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-446:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-413:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-406:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-464:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-467:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-425:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-415:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-431:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-465:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-463:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-420:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "SuccessSuccess\n",
      "Success\n",
      "\n",
      "errorSuccess\n",
      "SuccessSuccess\n",
      "\n",
      "SuccessSuccess\n",
      "\n",
      "\n",
      "errorSuccess\n",
      "\n",
      "SuccessSuccess\n",
      "\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "SuccessSuccess\n",
      "errorSuccess\n",
      "\n",
      "\n",
      "error\n",
      "Success\n",
      "SuccessSuccess\n",
      "\n",
      "SuccessSuccess\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-423:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-468:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-434:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-422:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-428:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-435:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-421:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-436:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-433:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-429:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-430:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-418:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-441:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-424:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-419:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-437:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-440:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-432:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-442:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-466:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-445:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-455:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-447:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-443:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-427:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-439:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-444:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-452:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Success\n",
      "SuccessSuccess\n",
      "Success\n",
      "Success\n",
      "\n",
      "SuccessSuccess\n",
      "\n",
      "errorSuccess\n",
      "SuccessSuccessSuccess\n",
      "SuccessSuccesserror\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "errorerrorSuccessSuccess\n",
      "\n",
      "\n",
      "\n",
      "error\n",
      "Success\n",
      "Success\n",
      "SuccessSuccess\n",
      "\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "SuccessSuccess\n",
      "Success\n",
      "\n",
      "SuccessSuccess\n",
      "\n",
      "Success\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-458:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-462:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-456:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-478:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-460:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-426:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-477:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-457:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-449:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-476:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-459:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-474:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-479:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-491:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-461:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-450:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-480:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-451:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-484:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-486:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-448:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-453:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-475:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-454:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-485:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-488:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-490:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-471:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-470:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-494:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-469:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-492:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-473:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "Exception in thread Thread-499:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-507:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-502:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-483:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-513:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-481:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-503:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-482:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-500:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-506:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-510:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-505:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-489:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-504:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-472:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-517:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-516:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Success\n",
      "Success\n",
      "SuccessSuccessSuccess\n",
      "\n",
      "\n",
      "SuccessSuccessSuccess\n",
      "\n",
      "Success\n",
      "\n",
      "error\n",
      "error\n",
      "SuccessSuccess\n",
      "\n",
      "Success\n",
      "Success\n",
      "Success\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-521:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-519:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-528:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-515:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-518:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-520:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-496:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-495:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-511:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-514:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SuccessSuccess\n",
      "\n",
      "SuccessSuccess\n",
      "\n",
      "Success\n",
      "Success\n",
      "error\n",
      "error\n",
      "Success\n",
      "Success\n",
      "Successerror\n",
      "\n",
      "Success\n",
      "Success\n",
      "errorSuccess\n",
      "\n",
      "errorerror\n",
      "error\n",
      "\n",
      "errorSuccess\n",
      "\n",
      "Success\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-497:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-509:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-508:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-498:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-524:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-512:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-522:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-523:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-525:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-487:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-526:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-501:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "<function TextIOWrapper.close()>"
      ]
     },
     "execution_count": 67,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Success\n",
      "Success\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-533:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-531:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Success\n",
      "Success\n",
      "SuccessSuccessSuccess\n",
      "error\n",
      "\n",
      "\n",
      "Success\n",
      "Success\n",
      "errorSuccess\n",
      "\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-532:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-534:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-537:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-493:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-536:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-535:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-576:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-578:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-530:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-527:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-577:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-573:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-540:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-582:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-529:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-543:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-554:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-553:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-545:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-549:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-558:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-550:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-557:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-561:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-547:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-588:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-585:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SuccessSuccess\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "SuccessSuccess\n",
      "Success\n",
      "SuccessSuccess\n",
      "\n",
      "\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "\n",
      "SuccessSuccess\n",
      "Success\n",
      "\n",
      "Success\n",
      "SuccessSuccess\n",
      "SuccessSuccess\n",
      "\n",
      "SuccesserrorSuccessSuccess\n",
      "Success\n",
      "\n",
      "Success\n",
      "\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "\n",
      "SuccessSuccess\n",
      "Success\n",
      "\n",
      "Success\n",
      "\n",
      "SuccessSuccess\n",
      "\n",
      "SuccessSuccess"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "Exception in thread Thread-541:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-584:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-542:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-586:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-596:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-604:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-552:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-559:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-602:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-544:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-587:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-560:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-562:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-603:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-548:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-599:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-556:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-598:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-555:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-551:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-597:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-601:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-583:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-546:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "\n",
      "Exception in thread Thread-566:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-570:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-600:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-595:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-568:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-571:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-572:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-567:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-569:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "SuccessSuccess\n",
      "\n",
      "error\n",
      "Success\n",
      "Success"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-539:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-538:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-574:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-594:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-580:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-589:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "Success\n",
      "SuccessSuccess\n",
      "\n",
      "Success\n",
      "Success\n",
      "Success\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-591:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-593:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-592:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-565:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-564:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-605:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "Exception in thread Thread-563:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "\n",
      "Exception in thread Thread-575:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-590:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Success\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-581:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n",
      "Exception in thread Thread-579:\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 926, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"C:\\Users\\Dongsun\\anaconda3\\lib\\threading.py\", line 870, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"<ipython-input-67-ce70a71c73a4>\", line 17, in append_line\n",
      "    time.sleep(1)\n",
      "NameError: name 'time' is not defined\n",
      "\n"
     ]
    }
   ],
   "source": [
    "import requests\n",
    "import csv\n",
    "import json\n",
    "import threading\n",
    "\n",
    "def append_line(url, targetList, line):\n",
    "    response = requests.get(url)\n",
    "    json_response = json.loads(response.text)\n",
    "    if(json_response['status'] == 'OK') :\n",
    "        line.append(json_response['candidates'][0]['formatted_address'])\n",
    "        line.append(json_response['candidates'][0]['geometry']['location'])\n",
    "        print('Success')\n",
    "    else : \n",
    "        line.append('error')\n",
    "        print('error')\n",
    "    targetList.append(line)\n",
    "    time.sleep(1)\n",
    "\n",
    "f = open('경상남도 창원시_불법주정차단속정보_20200831_수정.csv', 'r')\n",
    "rdr = csv.reader(f)\n",
    "index = 0\n",
    "lines = []\n",
    "for line in rdr:\n",
    "    index+=1\n",
    "    if index > 1 :\n",
    "        place = line[2]\n",
    "        url = 'https://maps.googleapis.com/maps/api/place/findplacefromtext/json?input=' + place +'&inputtype=textquery&fields=geometry,formatted_address&locationbias=circle:20000@35.2275496,128.6810085&key=AIzaSyDXwvMn4rRZ7e8yKR-gykRiO6dYQPJpOmU'\n",
    "        t1 = threading.Thread(target=append_line, args=(url, lines, line))\n",
    "        t1.start()\n",
    "    if index > 300:\n",
    "        break\n",
    "f.close\n",
    "\n",
    "nf = open('경상남도 창원시_불법주정차단속정보_20200831_결과.csv', 'w', newline='')\n",
    "wr = csv.writer(nf)\n",
    "wr.writerows(lines)\n",
    "nf.close\n",
    "print('complete!')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
